source/assets/js/components/fef-form-field.js

```diff
@@ -0,0 +1,34 @@
+import {DOM_CHANGED_EVENT} from '../classes/fef-dom-observer';
+
+$(window).on(DOM_CHANGED_EVENT, (e) => {
+    $('.radio-button, .checkbox').each((index, element) => {
+        new FefFormField($(element));
+    });
+});
+
+/**
+ * This component enriches radio buttons and checkboxes with A11y features
+ */
+export class FefFormField {
+
+    /**
+     * @param $element jQuery element
+     */
+    constructor($element) {
+        this.bindEvents($element);
+    }
+
+    /**
+     * @param $element jQuery element
+     */
+    bindEvents($element) {
+        // Enable checking radios by tabbing in + <enter>
+        $element.on('keypress', (e) => {
+            if (e.keyCode === 13) {
```

Comment:
Can we move the 13 to a constant in the to-be-defined Util class?

Reply:
Yes, I'll create a task for this